excel/src/main/java/tech/tablesaw/io/xlsx/XlsxReader.java

```diff
@@ -58,6 +58,7 @@ public static void register(ReaderRegistry registry) {
     registry.registerOptions(XlsxReadOptions.class, INSTANCE);
   }
 
+  // TODO Read by sheet name
```

Comment:
Let's leave out the TODO. There's tons of features we could add in the future and I don't know if putting a TODO for each one is the right way to track them

Reply:
Okay sure! I got busy with other work and so could not start this. Let me remove the TODO. By the way, this library is very useful. I used to work with python for data science but recently started using Java. And libraries like these help me focus on the core part rather than data loading. Please let me know on how else can I contribute to your roadmap. Will try to contribute as much as possible from my end.

Reply:
We don't really have much of a roadmap. I've just been adding things as I encounter a need for them. But I've slowed down a lot recently since most of the stuff I need the most is present now

Reply:
Somehow it is good. Maybe we can get some feature inspiration from pandas or matplotlib, just a small suggestion